flax/linen/module.py

```diff
@@ -1353,6 +1355,95 @@ def __call__(self, x):
     self.scope.put_variable(col, name, xs)
     return True
 
+  def tabulate(
```

Comment:
tabulate does not yet show up in the docs. You still need to add it to flax.linen.rst Note you can click "details" on the RTD action to see a preview of how the docs will look

Reply:
I think inspection/summarization makes sense